content/zh/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough.md

```diff
@@ -18,84 +18,106 @@ weight: 100
 <!-- overview -->
```

Comment:
The title has been changed;
There is now a `min-kubernetes-server-version: 1.23` in the frontmatter.